### PR TITLE
Ok1

### PR DESCRIPTION
**Pull request by hankwei-nextgen**
please merge ~

### CODE REVIEW

**Comment by hankwei-nextgen** [comment]
conflict resolved ~

**Comment by hankwei-nextgen** [comment]
> conflict resolved ~

yayaya

**Comment by hankwei-nextgen** [comment]
reopen ~

File: index.html [hunk 1]
```diff
@@ -3,15 +3,16 @@
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>dev</title>
```
Comment (HicksonWei) [inline index.html:6]: wtf?

File: index.html [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+		<title>hello</title>
 		<link rel="stylesheet" href="style.css" />
+		<script defer src="./index.js"></script>
```
Comment (HicksonWei) [inline index.html:8]: hello

File: index.js [hunk 3]
```diff
@@ -1 +1 @@
-console.log('hello')
```
Comment (HicksonWei) [inline index.js:1]: nonono

**Review by HicksonWei — COMMENTED** [review]
haha

**Pull request merged**